### PR TITLE
Feature/docspace 2.6 create filling form room

### PR DESCRIPTION
**Pull request by LolaChernysheva**

### CODE REVIEW

File: Documents/Documents/Code/Controllers/Documents/ASCDocumentsViewController.swift [hunk 1]
```diff
@@ -1985,6 +2009,17 @@ class ASCDocumentsViewController: ASCBaseViewController, UIGestureRecognizerDele
             }
         }
     }
+    
+    func duplicateRoom(room: ASCFolder) {
+        RoomSharingNetworkService().duplicateRoom(room: room) { result in
+            switch result {
+            case let .success(responce):
+                print("=====responce", responce)
```
Comment (chernyshevpavel) [inline Documents/Documents/Code/Controllers/Documents/ASCDocumentsViewController.swift:2017]: это можно убрать

File: Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingNetworkService.swift [hunk 2]
```diff
@@ -91,6 +91,22 @@ final class RoomSharingNetworkService: RoomSharingNetworkServiceProtocol {
             completion(.success(responce))
         }
     }
+    
+    func duplicateRoom(room: ASCFolder, completion: @escaping (Result<ASCFolder, Error>) -> Void) {
+        guard let roomId = Int(room.id) else { return }
```
Comment (chernyshevpavel) [inline Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingNetworkService.swift:96]: there is could be a string not Int only, change the model

File: Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingView.swift [hunk 3]
```diff
@@ -107,17 +123,22 @@ struct RoomSharingView: View {
...
-            if viewModel.sharedLinksModels.count < viewModel.linksLimit && viewModel.isSharingPossible {
-                Button {
-                    viewModel.createAddLinkAction()
-                } label: {
-                    Image(systemName: "plus")
-                        .foregroundColor(Asset.Colors.brend.swiftUIColor)
+        if viewModel.room.isFillingFormRoom {
+            Text(NSLocalizedString("Public link", comment: ""))
+        } else {
+            HStack {
```
Comment (chernyshevpavel) [inline Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingView.swift:131]: can we wrap it in a view var, sharedLinksRow for example?

Comment (LolaChernysheva) [reply Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingView.swift:131]: ✅

File: Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingNetworkService.swift (hunk 2 above)
Comment (LolaChernysheva) [reply Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingNetworkService.swift:96]: ✅

File: Documents/Documents/Code/Controllers/Documents/ASCDocumentsViewController.swift (hunk 1 above)
Comment (LolaChernysheva) [reply Documents/Documents/Code/Controllers/Documents/ASCDocumentsViewController.swift:2017]: ✅

**Review by chernyshevpavel — APPROVED** [review]

**Pull request merged**